Gems/Atom/RHI/Vulkan/Code/Source/RHI/CommandList.cpp

```diff
@@ -853,7 +853,12 @@ namespace AZ
                     shaderResourceGroup = shaderResourceGroupList.front();
                 }
                 
-                AZ_Assert(shaderResourceGroup, "Shader resource group in descriptor set index %d is null.", index);
+                if (shaderResourceGroup == nullptr)
+                {
+                    AZ_Assert(false, "Shader resource group in descriptor set index %d is null.", index);
+                    continue;
```

Comment:
Out of curiosity, were we hitting this assert?

Reply:
Yep, hair gem hits this when triggering multiviewport in the editor.